code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -157,40 +186,61 @@
 			owner.hud_used.move_intent.icon_state = "running"
 		owner.update_icons()
 
-	cancel_stealth()
+	if(pounce_break_stealth)
+		cancel_stealth()
 
 /// Callback for when a mob gets hit as part of a pounce
 /datum/action/ability/xeno_action/stealth/proc/mob_hit(datum/source, mob/living/M)
 	SIGNAL_HANDLER
 	if(M.stat || isxeno(M))
 		return
 	if(can_sneak_attack)
-		M.adjust_stagger(3 SECONDS)
-		M.add_slowdown(1)
+		var/datum/action/ability/activable/xeno/hunter_mark/assassin/mark = owner.actions_by_path[/datum/action/ability/activable/xeno/hunter_mark/assassin]
+		if(mark.marked_target == M)
+			to_chat(owner, span_xenodanger("We strike our death mark with a calculated pounce."))
+			M.adjust_stagger(6 SECONDS)
+			M.add_slowdown(2)
+		else
+			M.adjust_stagger(3 SECONDS)
+			M.add_slowdown(1)
 		to_chat(owner, span_xenodanger("Pouncing from the shadows, we stagger our victim."))
+	if(pounce_hit_break_stealth)
+		cancel_stealth()
+	else
+		can_sneak_attack = FALSE
+		addtimer(CALLBACK(src, PROC_REF(sneak_attack_cooldown)), HUNTER_POUNCE_SNEAKATTACK_DELAY)
 
 ///Special sneak attack when stealthed
 /datum/action/ability/xeno_action/stealth/proc/sneak_attack_slash(datum/source, mob/living/target, damage, list/damage_mod, list/armor_mod)
 	SIGNAL_HANDLER
 	if(!can_sneak_attack)
 		return
 
-	var/staggerslow_stacks = 2
+	var/staggerslow_stacks = 2 SECONDS
+	var/paralyzesecs = 1 SECONDS
 	var/flavour
 	if(owner.m_intent == MOVE_INTENT_RUN && ( owner.last_move_intent > (world.time - HUNTER_SNEAK_ATTACK_RUN_DELAY) ) ) //Allows us to slash while running... but only if we've been stationary for awhile
 		flavour = "vicious"
 	else
-		armor_mod += HUNTER_SNEAK_SLASH_ARMOR_PEN
-		staggerslow_stacks *= 2
+		armor_mod += sneak_attack_armor_pen
 		flavour = "deadly"
 
 	owner.visible_message(span_danger("\The [owner] strikes [target] with [flavour] precision!"), \
 	span_danger("We strike [target] with [flavour] precision!"))
-	target.adjust_stagger(staggerslow_stacks SECONDS)
+	var/datum/action/ability/activable/xeno/hunter_mark/assassin/mark = owner.actions_by_path[/datum/action/ability/activable/xeno/hunter_mark/assassin]
+	if(mark?.marked_target == target)
+		to_chat(owner, span_xenodanger("We strike our death mark with a [flavour], calculated strike."))
+		staggerslow_stacks *= 2
+		paralyzesecs *= 2
+	target.adjust_stagger(staggerslow_stacks)
 	target.add_slowdown(staggerslow_stacks)
-	target.ParalyzeNoChain(1 SECONDS)
+	target.ParalyzeNoChain(paralyzesecs)
 
-	cancel_stealth()
+	if(disable_on_mob_slash)
+		cancel_stealth()
+	else
```

Comment:
this is some pretty aids chainstun garbage tbh. Bro is going to be perma staggered, slowed for several years, and just chain stunned like 5 times.

Reply:
This is/was relating to the fact that assassin does not break stealth by attacking, so you chain them together repeatedly.

Reply:
You can only get 1 sneak attack per cast of stealth with the way it is right now, iirc. Since the stealth last 6 seconds, and it takes 3 seconds to ready a sneak attack, it's basically impossible to get 2 off.